### PR TITLE
update mpl token auth rules to the recent 1.18 solana

### PR DESCRIPTION
**Pull request by rwwwx**

### CODE REVIEW

**Comment by rwwwx** [comment]
As far as I understand `JS client` check was skipped because ci didn't detect any changes related to JS in this PR, so I think it's fine 🤔

**Review by StanChe — APPROVED** [review]
Great job, thanks! Let's wait for an approval from @blockiosaurus and merge it

**Review by blockiosaurus — APPROVED** [review]

**Pull request merged**
